### DISCUSSION TITLE
Should we pass subscribing challenges on the filter?

### DISCUSSION THREAD

**Discussion by resucutie** [ThioJoe/YT-Spammer-Purge#361] [General]
After a scan on [this](https://www.youtube.com/watch?v=D8mFvnVwHfU) Beluga video I've found multiple users with the name "1k challenge without any video", and so I was wondering if this is harmful or not. So I've started a discussion about if it is harmful in your opinion. Any opinions are accepted, even extreme ones
![image](https://user-images.githubusercontent.com/27961224/148694047-0524f8c2-93a9-48c9-a0d5-c287542543e4.png)

**Comment by ThioJoe** [comment]
They're against YouTube's terms of service as far as I know, because they all just spam comments. Even the original 100k Subs No Videos guy was banned.
